package.json

```
@@ -10,6 +10,9 @@
   "exports": {
     ".": {
       "import": "./dist/higlass.mjs"
+    },
+    "./dist/hglib.css": {
+      "import": "./dist/hglib.css"
     }
```

I forgot about this. I think we could keep compatibility with existing bundlers with:

```suggestion
    "./dist/*": "./dist/*"
```